README.md

```diff
@@ -13,17 +14,17 @@
 <img align="right" src="https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2FJDSherbert%2FUnreal-Engine-Gameplay-Module-Template%2Fhit-counter%2FREADME&count_bg=%2379C83D&title_bg=%23555555&labelColor=0E1128&title=🔍&style=for-the-badge">
 <!-- Header End --> 
 
------------------------------------------------------------------------
+</br>
 
 <a href="https://docs.unrealengine.com/5.3/en-US/"> 
   <img align="left" alt="Unreal Engine Template" src="https://img.shields.io/badge/Unreal%20Engine%20Template-black?style=for-the-badge&logo=unrealengine&logoColor=white&color=black&labelColor=black"> </a>
   
 <a href="https://choosealicense.com/licenses/unlicense/"> 
   <img align="right" alt="License" src="https://img.shields.io/badge/License%20:%20Unlicense-black?style=for-the-badge&logo=unlicense&logoColor=white&color=black&labelColor=black"> </a>
   
-<br></br>
+</br>
```

Comment:
In Github Markdown, there isn't a need to provide additional `<br>` it can just be referenced as `</br>` Opens and closes, though with it it requires that there is 1 space above and below on where it has been added.